test/e2e/all.spec.ts

```diff
@@ -0,0 +1,175 @@
+  })
+})
+
+async function doLogin(page: any) {
+  await page.goto('http://localhost:4000/login');
+  await page.getByRole('textbox', { name: 'Phone Number' }).click();
+  await page.getByRole('textbox', { name: 'Phone Number' }).fill('6475555555');
+  await page.getByRole('button', { name: 'Get Login Code' }).click();
+  await page.getByRole('textbox', { name: 'Authentication Code' }).click();
+  await page.getByRole('textbox', { name: 'Authentication Code' }).fill('123456');
+  await page.getByRole('button', { name: 'Sign in' }).click();
+}
+
+
+function getDatePlusDays(daysToAdd: number) {
+  const today = new Date();
+  const futureDate = new Date(today);
+  futureDate.setDate(today.getDate() + daysToAdd);
+
+  const year = futureDate.getFullYear();
+  const month = String(futureDate.getMonth() + 1).padStart(2, '0');
+  const day = String(futureDate.getDate()).padStart(2, '0');
+
+  return `${year}-${month}-${day}`;
+}
+
+async function createProgram(page: Page, programName: string) {
+  await page.goto('http://localhost:4000/programs');
+  await page.getByRole('link', { name: 'Programs' }).click();
+  await page.getByRole('link', { name: 'New Program' }).click();
+  await page.getByRole('textbox', { name: 'Name', exact: true }).click();
+  await page.getByRole('textbox', { name: 'Name', exact: true }).fill(programName);
+  await page.getByRole('textbox', { name: 'Campaign Blurb (please keep' }).click();
+  await page.getByRole('textbox', { name: 'Campaign Blurb (please keep' }).fill('This is a test program');
+  await page.getByRole('textbox', { name: 'About (internal description)' }).click();
+  await page.getByRole('textbox', { name: 'About (internal description)' }).fill('This is an internal description');
+  await page.getByRole('textbox', { name: 'Start Date' }).fill('2025-02-12');
+  await page.getByRole('checkbox', { name: 'Public' }).check();
+  await page.getByRole('checkbox', { name: 'Hide Pickup Address' }).check();
+  await page.getByRole('button', { name: 'Add Schedule' }).click();
+
+  await page.getByRole('textbox', { name: 'Photo Description' }).click();
+  await page.getByRole('textbox', { name: 'Photo Description' }).fill('1 Large Box');
+  await page.getByRole('textbox', { name: 'Contact Name' }).click();
+  await page.getByRole('textbox', { name: 'Contact Name' }).fill('Joe Cool');
+  await page.getByRole('textbox', { name: 'Contact Name' }).press('Tab');
+  await page.getByRole('textbox', { name: 'Contact Email' }).fill('joecool@gmail.com');
+  await page.getByRole('textbox', { name: 'Contact Email' }).press('Tab');
+  await page.getByRole('textbox', { name: 'Contact Phone' }).fill('6475555554');
+  await page.getByRole('button', { name: 'Save' }).click();
+
+
+  // once the program is created, we need to find it on the program page and EDIT it,
+  // because at this time there is no "add items" for part when creating a new program
+  // (only when editing it)
```

Comment:
I wonder if this is an intentional part of the New Program form, or if the ability to add items should be possible in that form.